src/inspect_ai/tool/_tools/_web_browser/_web_browser.py

```diff
@@ -56,146 +71,198 @@ def web_browser_click() -> Tool:
         """
-        return await web_browser_cmd("web_scroll", direction)
+        return await web_browser_cmd("web_type_submit", str(element_id), text)
 
     return execute
 
 
 @tool
-def web_browser_forward() -> Tool:
-    """Web Browser tool for navigating forward in the browser history.
-
-    Args:
-       sandbox (str): Name of sandbox that the web
-         browser is running within. Defaults to
-         "web_browser"
+def web_browser_type() -> Tool:
+    """Web Browser tool for typing into inputs.
 
     Returns:
-       Web browser forward navigation tool.
+       Web browser typing tool.
     """
 
-    async def execute() -> str:
-        """Navigate the web browser forward in the browser history.
+    async def execute(element_id: int, text: str) -> str:
+        """Type text into an input on a web browser page.
+
+        For example, to type "Norah" into the "First Name" search box on this page:
+
+        ```
+        [106] RootWebArea "My Profile" [focused: True, url: https://profile.example.com]
+          [305] link "My library" [url: https://profile.example.com/library]
+          [316] textbox "First Name" [focused: True, editable: plaintext, required: False]
+          [316] textbox "Last Name" [focused: True, editable: plaintext, required: False]
+        ```
+
+        You would use the following command:
+
+        web_browser_type(316, "Norah")
+
+        Note that the web_browser_type_submit tool is typically much more useful than the web_browser_type tool since it enters input and submits the form. You would typically only need to use the web_browser_type tool to fill out forms with multiple inputs.
+
+        Args:
+           element_id (int): ID of the element to type text into.
+           text (str): Text to type.
 
         Returns:
            Web accessibility tree of the visible elements of the web page. The element_id of each element is displayed in brackets at the beginning of the line.
         """
-        return await web_browser_cmd("web_forward")
+        return await web_browser_cmd("web_type", str(element_id), text)
 
     return execute
 
 
 @tool
-def web_browser_back() -> Tool:
-    """Web Browser tool for navigating back in the browser history.
+def web_browser_scroll() -> Tool:
+    """Web Browser tool for scrolling up or down one page.
 
     Returns:
-       Web browser back navigation tool.
+       Web browser scrolling tool.
     """
 
-    async def execute() -> str:
-        """Navigate the web browser back in the browser history.
+    async def execute(direction: str) -> str:
+        """Scroll the web browser up or down by one page.
+
+        Occasionally some very long pages don't display all of their content at once. To see additional content you can scroll the page down with:
+
+        web_browser_scroll("down")
+
+        You can then return to the top of the page with:
```

Comment:
Not familiar with the tools, but I suspect it would be accurate to say "You can then return to the previous scroll position with `web_browser_scroll("up")`